### DISCUSSION TITLE
what is the correct way to wire pa15(usb detection)

### DISCUSSION THREAD

**Discussion by fuzzbuzzbay999** [OpenI6X/opentx#269] [Q&A] [answered]
I mostly use my transmitter for sim racing, and recently upgraded from stock to openi6x since I needed ppm input for a project. 
However not having auto usb detection is a real pain.
![openi6x_hardware](https://user-images.githubusercontent.com/51093507/177063258-77426457-bbcb-4519-8dc7-fefdca238ceb.jpeg)
How is pa15 supposed to be wired to the usb data pins?

**Comment by fuzzbuzzbay999** [comment]
like does it matter what data pin I connect pa15 too?

**Comment by psicho100pro** [comment]
You need to use USB VBUS to PA15 pad.

**Reply by fuzzbuzzbay999** [reply]
I can't seem to find the usb voltage bus
![20220727_130723](https://user-images.githubusercontent.com/51093507/181363075-4d7eadb7-83f3-47f8-b4c2-c39453168e3f.jpg)
Am I looking for the wrong vbus?

**Comment by psicho100pro** [comment, marked as answer]
Simply connect USB V+ to PA15 through some step-down resistor, don't look for anything else,,,

**Reply by fuzzbuzzbay999** [reply]
Aight I think I got it, flysky didnt wire in the usb 5v bus, from the cable or trainer port so I wired in a little button that connects pa15 to 5v over 10k resistor. 
Press the button to connect

**Reply by Muhaiminul-Hasan** [reply]
mine works like a champ. thanks
![IMG202211301](https://user-images.githubusercontent.com/50764001/205274785-94a7cc51-2aaa-4bf4-89f8-985ddb95b7cd.jpg)

![IMG20221130183504](https://user-images.githubusercontent.com/50764001/204826685-acfd33d2-1888-4a5c-b09b-f2dcb0273548.jpg)
